docker/allensdk/Dockerfile

```diff
@@ -1,11 +1,29 @@
-# DOCKER-VERSION 1.11.1
+# DOCKER-VERSION 2.2.0.5
 #
 # docker build --tag alleninstitute/allensdk .
 # docker run -it alleninstitute/allensdk /bin/bash
 #
-FROM alleninstitute/anaconda_neuron_1370
+FROM continuumio/anaconda3
 
-MAINTAINER Tim Fliss <timf@alleninstitute.org>
```

Comment:
Instead of removing this section change it to labels as noted [here](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated). Have me as the maintainer as such `maintainer="isaak.willett@alleninstitute.org"`

Reply:
Should be fixed.